app/scenes/channel_members/channel_members_container.js

```diff
@@ -16,7 +16,8 @@ import {getProfilesInChannel} from 'service/actions/users';
 import ChannelMembers from './channel_members';
 
 function mapStateToProps(state) {
-    const currentChannelMemberCount = getCurrentChannelStats(state) && getCurrentChannelStats(state).member_count;
+    // Decrease the member count since we exclude the current user.
+    const currentChannelMemberCount = getCurrentChannelStats(state) && getCurrentChannelStats(state).member_count - 1;
```

Comment:
What if we show the correct member count but when checking if we need to load more we subtract the one